platform/pensando/dpu/debian/changelog

```diff
@@ -0,0 +1,5 @@
+dpu (1.0.0) UNRELEASED; urgency=low
```

Comment:
is dpu a docker package? why do we need debian packaging for this one?

Reply:
@ashwin-h , Can we delete this package and dpu folder?
For docker start:
files/build_templates/sonic_debian_extension.j2:43:DOCKER_CTL_SCRIPT="$DOCKER_CTL_DIR/docker.sh"
files/build_templates/sonic_debian_extension.j2:70:sudo cp $DOCKER_SCRIPTS_DIR/docker "$FILESYSTEM_ROOT/$DOCKER_CTL_SCRIPT"
files/build_templates/sonic_debian_extension.j2:75:    sudo chroot $FILESYSTEM_ROOT $DOCKER_CTL_SCRIPT start
files/build_templates/sonic_debian_extension.j2:854:    sudo chroot $FILESYSTEM_ROOT $DOCKER_CTL_SCRIPT stop

files/docker/docker

Reply:
Deleted dpu directory.